tokio-quiche/src/socket/capabilities.rs

```diff
@@ -268,8 +295,15 @@ impl<'s> SocketCapabilitiesBuilder<'s> {
     /// will only check their status. **If neither of them is enabled, the
     /// `PKTINFO` sockopts will cause errors when sending packets.**
     pub fn allows_nonlocal_source(&self) -> io::Result<bool> {
-        Ok(getsockopt(self.socket.as_raw_fd(), IpFreebind)? ||
-            getsockopt(self.socket.as_raw_fd(), IpTransparent)?)
+        Ok(getsockopt(&self.socket.as_fd(), IpFreebind)? ||
+            getsockopt(&self.socket.as_fd(), IpTransparent)?)
+    }
+
+    pub fn mark(&mut self) -> io::Result<()> {
```

Comment:
nit: should this be called recv_mark instead of just mark?

Reply:
Changed